src/common/reduction/matching/GapMatcher.cpp

```diff
@@ -1,4 +1,4 @@
-/* Copyright (C) 2018 European Spallation Source, ERIC. See LICENSE file */
+/* Copyright (C) 2018-2022 European Spallation Source, ERIC. See LICENSE file */
```

Comment:
Move to c++ comment while  editing
// Copyright (C) 2018 - 2022 European Spallation Source, ERIC. See LICENSE file

Reply:
Addressing in restyle branch